src/components/GaugeCard/GaugeCard.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+
+import { CARD_CONTENT_PADDING } from '../../constants/LayoutConstants';
+import { CardPropTypes, GaugeCardPropTypes } from '../../constants/PropTypes';
+import Card from '../Card/Card';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+// r value of the circle in SVG
+const radius = 30;
+// radius doubled plus stroke
+const gaugeSize = radius * 2 + 8;
+export const getStrokeDash = (value = 0) => {
+  // circumference of SVG.
+  const circum = 2 * Math.PI * radius;
+  // length of stroke to match percentage
+  return (value * circum) / 100;
+};
+
+export const getColor = (gauge, value) => {
+  let { color } = gauge;
+  let grade = '';
+  const comparisons = {
+    '<': (a, b) => a < b,
+    '>': (a, b) => a > b,
+    '=': (a, b) => a === b,
+    '<=': (a, b) => a <= b,
+    '>=': (a, b) => a >= b,
```

Comment:
Imagine how much more powerful the comparisons could be if we were allowed to supply our own custom comparsion functions.  Doing ranges would be breeze.

Reply:
I will address these changes and comments in our upcoming sprint. [Issue](https://github.com/IBM/carbon-addons-iot-react/issues/888) to track